doc-assets/node/18005-search-videos/18005.html

```diff
@@ -348,52 +265,62 @@ <h2 id="Quoted_search_terms">Quoted search terms</h2>
+    <pre class="line-numbers"><code class="language-http">https://cms.api.brightcove.com/v1/accounts/921483702001/videos?q=tags:"sea,mammal"</code></pre>
 
-    <aside class="bcls-aside bcls-aside--warning">Tags are a special case where you can search for multiple tags as a comma-delimited list - that is not true for other fields like name, description, etc. For other kinds of multiple terms, combine terms with <code>+</code> sign, like this: <code>q=tags:"bird"+name:"owl"</code>.</aside>
+    <aside class="bcls-aside bcls-aside--warning">Tags are a special case where you can search for multiple tags as a comma-delimited list - that is not true for other fields like name, description, etc. For other kinds of multiple terms, combine terms with <code>+</code> sign, like this: <code>q=tags:"bird"%20name:"owl"</code>.</aside>
   </section>
 
+  <!-- Custom Field Search -->
   <section class="bcls-section">
     <h2 id="custom_fields">Custom fields</h2>
 
     <p>You may search on any custom field that you have defined for your videos.</p>
 
-<pre class="line-numbers"><code class="language-http">q=my_field:foo
+    <pre class="line-numbers"><code class="language-http">q=my_field:foo
 q=my_field:"foo"</code></pre>
 
     <p class="BCL-aside">Note: all custom field values are treated as strings. For example, if you have a list-type custom field that can take the values <code>true</code> or <code>false</code>, search will look for those strings, not boolean values (in many programming languages, <code>1</code> and <code>0</code> can be used interchangeably with <code>true</code> and <code>false</code> as boolean values, but searching on <code>q=my_boolean_field:1</code> will not return videos that have <code>my_boolean_field</code> set to <code>true</code>).</p>
 
     <p>Example: This request returns videos which have a value of <strong>animal</strong> in the <code>subject</code> custom field.</p>
 
-<pre class="line-numbers"><code class="language-http">https://cms.api.brightcove.com/v1/accounts/921483702001/videos?q=subject:animal</code></pre>
+    <pre class="line-numbers"><code class="language-http">https://cms.api.brightcove.com/v1/accounts/921483702001/videos?q=subject:animal</code></pre>
   </section>
 
+  <!-- Date Search -->
   <section class="bcls-section">
     <h2 id="date_ranges">Date ranges</h2>
 
     <p>If you are searching on a date field, you can specify a specific date or a range of dates, using two periods to separate the start and end dates (<code>q=updated_at:2018-01-01..2018-02-01</code>).</p>
 
     <p>This will search for all videos with an <code>updated_at</code> value between Aug 1, 2012 and October 8, 2012. Here we are specifying each date in UTC format.</p>
 
-<pre class="line-numbers"><code class="language-http">q=updated_at:2012-08-01T00:00:00Z..2012-10-08T00:00:00Z</code></pre>
+    <pre class="line-numbers"><code class="language-http">q=updated_at:2012-08-01T00:00:00Z..2012-10-08T23:59:99Z</code></pre>
```

Comment:
The description was incorrect due to the timestamps on the dates.  Maybe we should just remove the time component?

Reply:
if you dropped the time component, the search with the new/adjusted time would match the search without the time component? (might be worth including that example as well?)

Reply:
Good point.  And its worth verifying that as well.

Reply:
And I mistook the seconds for milliseconds.  That should be 59.